src/schemas/settings/v1/index.ts

```diff
@@ -86,23 +87,41 @@ export const RecorderSettingsSchema = z
     }
   })
 
-const WindowStateSchema = z.object({
+export const WindowStateSchema = z.object({
   x: z.number().int(),
   y: z.number().int(),
   width: z.number().int(),
   height: z.number().int(),
   isMaximized: z.boolean(),
 })
 
-const AppearanceSchema = z.object({
+export const AppearanceSchema = z.object({
   theme: z.union([z.literal('light'), z.literal('dark'), z.literal('system')]),
 })
 
 export const AppSettingsSchema = z.object({
-  version: z.string(),
+  version: z.literal('1.0'),
   proxy: ProxySettingsSchema,
   recorder: RecorderSettingsSchema,
   windowState: WindowStateSchema,
   usageReport: UsageReportSettingsSchema,
   appearance: AppearanceSchema,
 })
+
+// Migrate settings to the next version
+export function migrate(
+  settings: z.infer<typeof AppSettingsSchema>
+): v2.AppSettings {
```

Comment:
Every schema should export a `migrate` function that takes the current version and returns the next.